text/0101-revamping-the-sdk-performance-api.md

```diff
@@ -0,0 +1,268 @@
+| `trace_id`        | Trace ID of the span. Format is identical between Sentry and OpenTelemetry                                                                                                                            | String          |
+| `span_id`         | Span ID of the span. Format is identical between Sentry and OpenTelemetry                                                                                                                             | String          |
+| `parent_span_id`  | Parent span ID of the span. Format is identical between Sentry and OpenTelemetry                                                                                                                      | String          |
+| `end_timestamp`   | Timestamp when span was finished. Some SDKs alias this to `end_timestamp` but convert it to `timestamp` when serializing to send to Sentry.                                                           | String \| Float |
+| `start_timestamp` | Timestamp when span was created.                                                                                                                                                                      | String \| Float |
+| `op`              | Operation type of the span. [Standardized by Sentry spec](https://develop.sentry.dev/sdk/performance/span-operations/)                                                                                | String          |
+| `status`          | Status of the span. Sentry maps status to HTTP status codes, while OpenTelemetry has a fixed set of status'                                                                                           | String          |
+| `tags`            | Custom tags for this span.                                                                                                                                                                            | Object          |
+| `data`            | Arbitrary additional data on a span, like `extra` on the top-level event. We maintain [conventions for span data keys and values](https://develop.sentry.dev/sdk/performance/span-data-conventions/). | Object          |
+
+As you can see, the fields on the transaction/span differ in a few ways, the most noteable of which is that transactions have `name` while spans have `description`. This means that spans and transactions are not interchangeable, and users have to know the difference between the two.
+
+In addition, user's have the burden to understand the differences between `name`/`description`/`operation`. `operation` in particular can be confusing, as it overlaps with transaction `name` and span `description`. In addition, `operation` is not a required field, which means that it is not clear what the default value should be.
```

Comment:
IIRC `operation` was always required in our data model.

Reply:
Not sure why it's a mandatory field in [Mobile](https://github.com/getsentry/sentry-java/blob/98fb9c76b168e19a601f82e878a5d539c1698e1a/sentry/src/main/java/io/sentry/ISpan.java#L17) then, either it was documented like that or is an oversight on our end, thanks for the link.